recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: 64f3551e5a2f8497399d82af3076b6a33bf1bc95fc46bbcabe66442db366f453
 
 build:
-  number: 1002
+  number: 1004
```

Comment:
why skip 1003 ?

Reply:
sorry, I had a local build of 1003. Changing this now.